src/NumericalAlgorithms/LinearAlgebra/FindEigenvalues.hpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/// \cond
+class ComplexDataVector;
+class ComplexMatrix;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Find the eigenvalues of a matrix
+ *
+ * This function uses the lapack routine zgeev
+ * (http://www.netlib.org/lapack/explore-3.1.1-html/zgeev.f.html) to
+ * solve the eigenvalue problem \f$A v_a = \lambda_a v_a \f$ for the
+ * eigenvalues \f$\lambda_a\f$.
+ */
+void find_eigenvalues(gsl::not_null<ComplexDataVector*> eigenvalues,
```

Comment:
I suggest you just use Blaze directly to create the matrix and compute its eigenvalues: https://bitbucket.org/blaze-lib/blaze/wiki/Matrix%20Operations#!eigenvalueseigenvectors

Reply:
Convenient.  Dropped the first commit because blaze already did it.